nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp

```diff
@@ -140,6 +140,11 @@ class NavfnPlanner : public nav2_core::GlobalPlanner
 
   // Whether to use the astar planner or default dijkstras
   bool use_astar_;
+
+  // Subscription for parameter change
+  rclcpp::AsyncParametersClient::SharedPtr parameters_client_;
+  rclcpp::Subscription<rcl_interfaces::msg::ParameterEvent>::SharedPtr parameter_event_sub_;
+  void on_parameter_event_callback(const rcl_interfaces::msg::ParameterEvent::SharedPtr event);
```

Comment:
Please add doxygen for this, if you pull in `main`, `navfn` has now complete doxgyen coverage